Python/sysmodule.c

```diff
@@ -819,6 +819,32 @@ sys_exception_impl(PyObject *module)
     Py_RETURN_NONE;
 }
 
+/*[clinic input]
+sys._set_exception
+    exception: object
+
+Set the current exception.
+
+Subsequent calls to sys.exception()/sys.exc_info() will return
+the provided exception until another exception is raised in the
```

Comment:
until another exception is caught (rather than raised).

Reply:
fixed the wording